src/project-2.js

```diff
@@ -3,6 +3,12 @@
 const getBiggest = (x, y) => {
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
+  if (x > y) {
+    return x;
+  } else if (y > x) {
+    return y;
+  }
+  return x;
 };
 
 const greeting = (language) => {
```

Comment:
If your base case matches one of your cases, you can just drop the `else if (language === 'English')`.
I'd also suggest using switch cases to reduce code size.
Lookup table would be fastest. 
https://jsperf.com/if-switch-lookup-table/10